src/embedding.rs

```diff
@@ -296,7 +296,7 @@ where
             self.dimension, entities_count
         );
 
-        mmap.flush();
+        mmap.flush().ok();
```

Comment:
Maybe instead of `ok` for now we can log error or add `expect`. These mmap errors are unrecoverable.